core/domain/email_manager.py

```diff
@@ -37,6 +37,20 @@
 def log_new_error(*args, **kwargs):
     logging.error(*args, **kwargs)
 
+NOTIFICATION_EMAIL_LIST_SCHEMA = {
+    'type': 'list',
+    'items': {
+        'type': 'unicode'
+    },
+    'validators': [{
+        'id': 'has_length_at_most',
+        'max_value': 5
+    }, {
+        'id': 'is_valid_email_list',
```

Comment:
Can this validator go in 'items', and turn into 'is_valid_email'?

I.e.

    'items': {
        'type': 'unicode',
        'validators': [{
            'id': 'is_valid_email',
        }]
    }

Reply:
done